vulnerabilities/package_managers.py

```diff
@@ -489,3 +509,75 @@ async def fetch(self, pkg, session):
             pass
 
         self.cache[pkg] = versions
+
+
+class GoproxyVersionAPI(VersionAPI):
+
+    package_type = "golang"
+    pkg_mappings = {}
+
+    @staticmethod
+    def trim_package_path(pkg: str) -> Optional[str]:
```

Comment:
Maybe rename the method and parameters that the input `pkg` is actually a http url